### ISSUE TITLE
PEDLA: predicting enhancers with a deep learning-based algorithmic framework

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#20]
https://dx.doi.org/10.1101/036129

**Comment by gwaygenomics** [comment]
published doi: http://dx.doi.org/10.1038/srep28517

**Comment by gwaygenomics** [comment]
## Goal

Predict enhancers using heterogeneous genomic data across different cell types

## Biology

Use histone mod, TFs, chromatin accessibility, DNA methylation, conservation, etc. to train a neural network to extract the probability of an enhancer at given genomic locations. These probabilities are then used in a Hidden Markov Model to predict enhancer states.

## Computational Aspects

* Significant amount of preprocessing of heterogeneous features 
  * RPKM-like measurement for each feature at 200 base pair bins
  * 0-1 normalized
* Gold standard enhancers defined based on H3K27ac peaks
* Unsupervised pre-training with deep belief network
  * Weights of DBN used to initialize deep neural network
* DNN outputs probability of enhancer given the location
  * Output probability is used in subsequent HMM to predict enhancer states
  * More info in [supplementary figure S1](http://www.nature.com/article-assets/npg/srep/2016/160622/srep28517/extref/srep28517-s1.pdf)
* Validation defined as enhancer overlap with p300, NANOG, OCT4, SOX2 binding sites

## Pros vs. Cons

# Pros
* After extensive preprocessing it seems to be a reasonable way of integrating heterogeneous features
* Extensive performance metric reporting
* Very nice comparison of alternative approaches (all machine learning based)
* Nice discussion of previous limitations in predicting enhancers and how PEDLA overcomes them
  * Low training samples
  * difficult to integrate heterogenous data
  * there is high class imbalance (PEDLA corrects for by normalizing by class imbalance probability in HMM)

# Cons
* Questionable gold standard and validation decisions
  * Gold standard labeling is one of the most difficult steps for enhancer classifiers but difficult to compare methods
 * 200 base pair bins - if any prediction overlaps an H3K27ac peak anywhere, is it considered to have successfully identify an enhancer?
* PEDLA doesn't _really_ work across different tissues
  * They claim it does but it really trains on one and then uses weights in a new training model on a different cell type/tissue
* In discussion: "...these studies inspired us to further improve PEDLA through the incoporation of 3D structure information."
  * No 3D structure features used

**Comment by cgreene** [comment]
Tagged as study. Closing for now. @gwaygenomics : if you want to highlight this example in the 'study' section, feel free to bring it in!
